contracts/ColonyFunding.sol

```diff
@@ -381,18 +384,16 @@ contract ColonyFunding is ColonyStorage, PatriciaTreeProofs {
     return (payout.tokenAddress, reward);
   }
 
-  function updateTaskPayoutsWeCannotMakeAfterPotChange(uint256 _id, address _token, uint _prev) internal {
-
+  function updateTaskPayoutsWeCannotMakeAfterPotChange(uint256 _id, address _token, uint _amount) internal {
     Task storage task = tasks[_id];
     uint totalTokenPayout = getTotalTaskPayout(_id, _token);
-    uint tokenPot = fundingPots[task.fundingPotId].balance[_token];
+    uint currentBalance = fundingPots[task.fundingPotId].balance[_token];
+    uint newBalance = add(currentBalance, _amount);
 
-    if (_prev >= totalTokenPayout) {                  // If the old amount in the pot was enough to pay for the budget
-      if (tokenPot < totalTokenPayout) {              // And the new amount in the pot is not enough to pay for the budget...
-        task.payoutsWeCannotMake += 1;                // Then this is a set of payouts we cannot make that we could before.
-      }
-    } else {                                          // If this 'else' is running, then the old amount in the pot could not pay for the budget
-      if (tokenPot >= totalTokenPayout) {             // And the new amount in the pot can pay for the budget
+    // It is not possible to defund a pot below the budget, so we don't check for that here.
```

Comment:
Mm the main argument I'd make for removing is that it reduces code coverage.

Reply:
Code coverage is a tool!

Reply:
I mean yes and no, when your build fails because of code coverage then it's a spec.

Reply:
Although I'm realizing that after merging in Payments that invariant no longer holds since we _can_ defund payments below the payout. So much ado about nothing.